src/lib/plugin_apis/lvm.api

```diff
@@ -136,6 +136,7 @@ void bd_lvm_pvdata_free (BDLVMPVdata *data) {
     g_free (data->pv_name);
     g_free (data->pv_uuid);
     g_free (data->vg_name);
+    g_free (data->vg_uuid);
```

Comment:
Please also update the copy function here.

Reply:
Done.